contracts/Nexus.sol

```diff
@@ -306,6 +320,10 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
         return _ACCOUNT_IMPLEMENTATION_ID;
     }
 
+    function setRegistry(IERC7484 newRegistry, address[] calldata attesters, uint8 threshold) external onlyEntryPointOrSelf {
+        _configureRegistry(newRegistry, attesters, threshold);
```

Comment:
at which point SA is made to call

function trustAttesters(
        uint8 threshold,
        address[] memory attesters // deliberately using memory to allow sorting and uniquifying
    )
    
  ?
  
That's an optional step with initcode approach we talked about right with specific factory when deploying the account?

Reply:
yes, this could be called during the initialization by the factory (eg see bootstrap) or after an account has been created already